src/ecmult_gen_compute_table_impl.h

```diff
@@ -85,6 +84,7 @@ static void secp256k1_ecmult_gen_compute_table(secp256k1_ge_storage* table, cons
     for (block = 0; block < blocks; ++block) {
         size_t index;
         for (index = 0; index < points; ++index) {
+            CHECK(!secp256k1_ge_is_infinity(&prec[block * points + index]));
```

Comment:
note: the precompute binaries are built with `-DVERIFY` (it's nevertheless okay to keep this here, I guess)